libs/clients/vmst/src/clientConfig.yaml

```diff
@@ -491,8 +489,8 @@ components:
           type: integer
           format: int32
         transferableMonths:
-          type: integer
-          format: int32
+          type: number
+          format: float
```

Comment:
I updated the client config and this was in the new config, although unrelated to this PR.